public/app/core/services/SidecarService.ts

```diff
@@ -2,12 +2,23 @@ import { BehaviorSubject } from 'rxjs';
 
 import { config, locationService } from '@grafana/runtime';
 
+interface Options {
+  locationStorageKey?: string;
```

Comment:
I think this should be rather something like `localStorageKey`

Reply:
Yeah weird typo I guess :D